underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -170,18 +186,7 @@ public OutputStream create(String path, CreateOptions options) throws IOExceptio
       return null;
     }
 
-    final String strippedPath = CommonUtils.stripPrefixIfPresent(path,
-        Constants.HEADER_SWIFT);
-    // TODO(adit): remove special handling of */_SUCCESS objects
```

Comment:
so the special handling of `*/_SUCCESS` is removed in this PR?  can you please explain the reason behind?